packages/twenty-front/src/modules/command-menu/pages/page-layout/components/CommandMenuPageLayoutIframeSettings.tsx

```diff
@@ -74,21 +69,6 @@ export const CommandMenuPageLayoutIframeSettings = () => {
 
   return (
     <>
-      <SidePanelHeader
```

Comment:
I'm unsure where you moved this logic. I'm worried it might make the DX more complicated.